WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1354,6 +1355,21 @@ private void showPublishConfirmationOrUpdateIfNotLocalDraft() {
         }
     }
 
+    private void showGutenbergInformativeDialog() {
+        // Show the GB informative dialog on editing GB posts
+        if (!mIsNewPost && !AppPrefs.isGutenbergInformativeDialogDisabled()) {
+            final PromoDialog asyncPromoDialog = new PromoDialog();
+            asyncPromoDialog.initialize(TAG_GB_INFORMATIVE_DIALOG,
+                    getString(R.string.dialog_gutenberg_informative_title),
+                    mPost.isPage() ? getString(R.string.dialog_gutenberg_informative_description_page)
+                    : getString(R.string.dialog_gutenberg_informative_description_post),
+                    getString(org.wordpress.android.editor.R.string.dialog_button_ok));
+
+            asyncPromoDialog.show(getSupportFragmentManager(), TAG_GB_INFORMATIVE_DIALOG);
+            AppPrefs.setGutenbergInformativeDialogDisabled(true);
+        }
```

Comment:
What about adding some tracking here? basically we can reuse 
```            
PostUtils.trackGutenbergDialogEvent(
                    AnalyticsTracker.Stat.GUTENBERG_INFORMATIVE_DIALOG_SHOWN, post, site);
``` 
 and add the event `AnalyticsTracker.Stat.GUTENBERG_INFORMATIVE_DIALOG_SHOWN` to the stats enum